### ISSUE TITLE
fashionclip model returning null vallues

### ISSUE THREAD

**Issue by khayreali** [abdullahtarek/basketball_analysis#1]
The 'patrickjohncyh/fashion-clip model' used in team assignment null values in the logits output, causing incorrect team classifications.

I am not sure exactly the issue here, but argmax is defaulting to 0. My guess is the model isn't calculating the confidence scores as expected.

Easy workaround for me was to use OpenAI's CLIP model. It worked well enough.
